### ISSUE TITLE
Default options i.e. Host *

### ISSUE THREAD

**Issue by QCU87Z** [voidquark/voidify#1]
Hey,

Enjoy some of your ansible roles, looking at this project as well. One thing I have in my ssh configuration normally is default options using Host *

i.e.

```
Host *
  User ash
  StrictHostKeyChecking false

Host dev
  Hostname ws.dev.example.com
  IdentityFile ~/id_ed25519_dev

Host lb
  Hostname lb.example.com
```

Is that something this could support? So I dont need to declear the some fields in the yaml?
The Env and fuzzy search is great though

Cheers,
Ash

**Comment by voidquark** [comment]
Hello,

This configuration can be achieved without modifying `voidify`. Instead, you can use the `Include` directive, which has been available since OpenSSH version 7.3.

For instance, create an inventory file named `inventory.yml` for voidify. However, avoid specifying the host with `*` directly, as it is not valid YAML:

```yaml
*:
  User ash
  StrictHostKeyChecking false
```

If you add the above content to the inventory file, it will fail.

Here's the correct approach:

1. Create a standard inventory file, for example, `inventory.yml`.

2. Create the following file in `~/.ssh/config`
```bash
Include ~/.ssh/generated_config

Host *
  User ash
  StrictHostKeyChecking false
```

3. Run voidify
```bash
voidify -c inventory.yml -o ~/.ssh/generated_config
```

Now it should work as expected.

**Comment by QCU87Z** [comment]
Yeah that works well. Just tested on my set up.
Never needed Include so didn't think about it.

Cheers
